### PR TITLE
EB-1470 (Edit Driver)

### PR DESCRIPTION
**Pull request by maartenhemmes**

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -1,30 +1,31 @@
 Arguments: 
-  /Users/vitalii/.nvm/versions/node/v6.11.3/bin/node /usr/local/Cellar/yarn/1.5.1_1/libexec/bin/yarn.js test -t should get true value for export file
```
Comment (maartenhemmes) [inline yarn-error.log]: @vdanylov please resolve merge conflicts

File: src/data/customer/__test__/driver.test.js [hunk 2]
```diff
@@ -414,3 +415,64 @@ function generateDisplayName(size) {
 
   return text;
 }
+
+// TODO check updateDriverAsync tests
```
Comment (maartenhemmes) [inline src/data/customer/__test__/driver.test.js]: rebase branch from CreateDriver and run the tests @vdanylov

File: src/data/Endpoint.js [hunk 3]
```diff
@@ -46,6 +46,7 @@ const endpoints = {
     OPTIMIZE_ROUTE: `${laravelUrlPrefix}/${version}/customer/route/optimization`,
     ROUTE_LOCATION: `${laravelUrlPrefix}/${version}/customer/route/{0}/route-location`,
     DRIVER_SCHEDULE: `${laravelUrlPrefix}/${version}/customer/driver-schedule/{0}`,
+    DRIVER_UPDATE: `${laravelUrlPrefix}/${version}/customer/driver/{0}`,
```
Comment (maartenhemmes) [inline src/data/Endpoint.js]: Shouldn't this be the same endpoint as +    CUSTOMER_DRIVER: `${laravelUrlPrefix}/${version}/customer/driver`,
Only a different method? @vdanylov

File: src/data/customer/Driver.js [hunk 4]
```diff
@@ -171,19 +172,120 @@ export const getDriversAsync = async (filterObject = {}, token) => {
...
+        color,
+      },
+      bank: {
+        branchCode,
+        code,
+        name,
+        accountNumber,
+      },
+    };
+    // TODO check if camelToSnake doesn't not apply to array
```
[the remark this answers is not in the document]
Comment (vdanylov) [reply src/data/customer/Driver.js]: @maartenhemmes this comments refers to case when array lays at the first level but for our object we have to use depth of 2, and then array of first level would be snaked

File: src/data/Endpoint.js (hunk 3 above)
Comment (vdanylov) [reply src/data/Endpoint.js]: @maartenhemmes according documentation, no.
screen shot 2018-05-11 at 11 49 29

File: src/data/customer/Driver.js (hunk 4 above)
[the remark this answers is not in the document]
Comment (vdanylov) [reply src/data/customer/Driver.js]: I meant this problem
screen shot 2018-05-11 at 12 08 28

[the remark this answers is not in the document]
Comment (vdanylov) [reply src/data/customer/Driver.js]: @maartenhemmes this function makes from array keyed object

File: src/data/customer/__test__/Config.js [hunk 5]
```diff
@@ -237,6 +237,104 @@ const CONFIG = {
       },
     ],
   },
+  driverEdit: {
+      id: 10714,
```
Comment (maartenhemmes) [inline src/data/customer/__test__/Config.js]: @vdanylov what are we supposed to do when the data changes in the database?

File: src/data/customer/__test__/driver.test.js [hunk 6]
```diff
@@ -407,3 +410,81 @@ function generateDisplayName(size) {
...
```
Comment (maartenhemmes) [inline src/data/customer/__test__/driver.test.js]: should remove this comment @vdanylov

**Pull request closed without merging**